gbpservice/nfp/service_plugins/vpn/drivers/nfp_vpnaas_driver.py

```diff
@@ -70,6 +75,11 @@ def ipsec_site_conn_deleted(self, context, ipsec_site_conn_id):
         plugin = self.driver.service_plugin
         plugin._delete_ipsec_site_connection(context, ipsec_site_conn_id)
 
+    def vpnservice_deleted(self, context, **kwargs):
```

Comment:
Don't see vpnservice_deleted callback in VPN agent code of configurator. Why is this added?